l1-contracts/test/prover-coordination/ProofCommitmentEscrow.t.sol

```diff
@@ -13,51 +13,45 @@ import {TestERC20} from "../TestERC20.sol";
 // solhint-disable comprehensive-interface
 
 contract TestProofCommitmentEscrow is Test {
-  ProofCommitmentEscrow internal _escrow;
-  TestERC20 internal _token;
+  // solhint-disable-next-line var-name-mixedcase
+  ProofCommitmentEscrow internal ESCROW;
+  // solhint-disable-next-line var-name-mixedcase
+  TestERC20 internal TOKEN;
   address internal prover;
   uint256 internal depositAmount;
 
-  modifier setup() {
-    _token = new TestERC20();
-    _escrow = new ProofCommitmentEscrow(_token, address(this));
-    _;
-  }
-
   modifier setupWithApproval(address _prover, uint256 _depositAmount) {
-    _token = new TestERC20();
-    _escrow = new ProofCommitmentEscrow(_token, address(this));
-
-    _token.mint(_prover, _depositAmount);
+    TOKEN.mint(_prover, _depositAmount);
     vm.prank(_prover);
-    _token.approve(address(_escrow), _depositAmount);
+    TOKEN.approve(address(ESCROW), _depositAmount);
 
     prover = _prover;
     depositAmount = _depositAmount;
     _;
   }
 
+  function setUp() public {
+    TOKEN = new TestERC20();
+    ESCROW = new ProofCommitmentEscrow(TOKEN, address(this));
+  }
+
   function testDeposit() public setupWithApproval(address(42), 100) {
```

Comment:
Slightly annoying here, but if the input is always the `_prover` and it will be the `prover` value it don't seem useful to provide as input. Also, if always having the same inputs might as well remove them from here.